tartufo.toml

```
@@ -3,3 +3,63 @@ exclude-paths = ".tartufo-excludes"
 json = false
 regex = true
 entropy = true
+exclude-signatures = [
+    # High entropy findings for images and keys used in documentation
+    "13c9d46245326e1995825a10ef733a7971bff3d293d58afa9ab16fb3770be8fb",
+    "5c26481a67a11bad0844db1cb4f5107e2ff2c8ccac4d77e27ebead916f0148fa",
+    "ae0ba571291755ac449e4fc6eb110e33faaf150f95b94a9ded06a925b07b79d0",
+    "bcd4bc83e3131567f85a21619ff810b7c8d8b01c7fa2fb2fa26b950b1ce526ce",
+    "23092a312f381b03347247bd874618b71df0069d42f5fa5343816fcee1cd1421",
+    "68aca477af868994b50620c412178789b05f70e34a4d2cc74694ab91f423397d",
+    "2889354774c90341ab3f702d8fe5f41e9ba202174b149d717d633b4093f9a85b",
+    "6e5fdaf1037c89aaff33729a6ed74131786816cc5d2aa81cbf5f235a93cfa4b8",
+    "a3fec2f6b84270361f3b24854ef8de5af2d757263c80e61180ed6ad8b9040c22",
+    "1ddecb6f7fe1f06d7c0dbbbce40ec3de3054e45b78be3beec02e47e0a7c34de8",
+    "004359c84fb457b50fb329f4ba31563e7260e55d08794a66e48969f03a54b8ac",
+
+    # High entropy findings (dummy encryoted payloads, fake keys, method names, etc.) in Java Tests
+    "6aecdbffd5066a9a5e366e2587cc6b8ca420d0d3600212039c7819824a5cf5fd",
+    "db4e5dccdd961ff01677023444541035042cc6de3e24282e67924719d35dfa58",
+    "d5c65da2e2ddce6a92cd94567fb3074f35cfd439b104a59f48b17eaed25b0264",
+#    "c9dcce093cdfb3ac2a36e4965c99e2158829db17d5ed9d048f9e4c576c1ad022",
+#    "de10bc1f76e947229b1776fa57e77f0202dc8d93ea63d0095e176a88e6bc1ff0",
+#    "b86d3814dca4cf16040fcab622aa7f2a64580f76c8d53375b856f9b9bfc1268f",
```

```suggestion
    "c9dcce093cdfb3ac2a36e4965c99e2158829db17d5ed9d048f9e4c576c1ad022",
    "de10bc1f76e947229b1776fa57e77f0202dc8d93ea63d0095e176a88e6bc1ff0",
    "b86d3814dca4cf16040fcab622aa7f2a64580f76c8d53375b856f9b9bfc1268f",
```
Running `tartufo scan-local-repo .` against this branch reports these sigs. Were they commented out for testing maybe?